gitmine/commands/get.py

```diff
@@ -122,6 +122,12 @@ def print_prs(prs: List[Mapping[str, Any]], color: bool) -> None:
             )
         )
 
+    reverse_bool = True if asc else False
+    projects = {
+        project: sorted(prs, key=lambda pr: pr.elapsed_time, reverse=reverse_bool)
+        for project, prs in projects.items()
+    }
```

Comment:
Interesting to see this. Never really used dictionary comprehension before. cool stuff